src/speed-dial/index.tsx

```diff
@@ -34,10 +66,36 @@ export const Action = actionFactory(({ properties, children, middleware: { theme
 			classes={classes}
 			variant={variant}
 		>
-			{children()}
+			{icon}
 		</FloatingActionButton>
 	);
 
+	if (label) {
+		let orientationClass: string | undefined;
+		switch (labelOrientation) {
+			case LabelOrientation.top:
+				orientationClass = themedCss.labelTop;
+				break;
+			case LabelOrientation.right:
+				orientationClass = themedCss.labelRight;
+				break;
+			case LabelOrientation.bottom:
+				orientationClass = themedCss.labelBottom;
+				break;
+			case LabelOrientation.left:
+			default:
+				orientationClass = themedCss.labelLeft;
+				break;
+		}
+
+		return (
+			<div classes={[themedCss.labelContainer, orientationClass]}>
+				<label classes={themedCss.label}>{label}</label>
```

Comment:
I think this might need to use an aria attribute or `for` to link to the fab's `id`